build error

### WorldEdit Version

latest

### WorldGuard Version

latest

### Platform Version

latest

### Confirmations

- [x] I am using the most recent Minecraft release.
- [x] I am using a version of WorldEdit compatible with my Minecraft version.
- [x] I am using a version of WorldGuard compatible with my Minecraft version.
- [x] I am using the latest or recommended version of my platform software.
- [x] I am NOT using a hybrid server, e.g. a server that combines Bukkit and Forge. Examples include Arclight, Mohist, and Cardboard.
- [x] I am NOT using a fork of WorldEdit, such as FastAsyncWorldEdit (FAWE) or AsyncWorldEdit (AWE)

### Bug Description

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':worldguard-bukkit:compileJava'.
> Could not resolve all dependencies for configuration ':worldguard-bukkit:compileClasspath'.
   > Could not resolve io.papermc.paper:paper-api:1.21.4-R0.1-SNAPSHOT.
     Required by:
         project :worldguard-bukkit
      > Module 'io.papermc.paper:paper-api' has been rejected:
           Cannot select module with conflict on capability 'org.spigotmc:spigot-api:1.21.4-R0.1-SNAPSHOT' also provided by [org.spigotmc:spigot-api:1.21.4-R0.1-SNAPSHOT(compile)]
   > Could not resolve org.spigotmc:spigot-api:1.21.4-R0.1-SNAPSHOT.
     Required by:
         project :worldguard-bukkit
      > Module 'org.spigotmc:spigot-api' has been rejected:
           Cannot select module with conflict on capability 'org.spigotmc:spigot-api:1.21.4-R0.1-SNAPSHOT' also provided by [io.papermc.paper:paper-api:1.21.4-R0.1-SNAPSHOT(apiElements)]

* Try:


### Expected Behavior

latest

### Reproduction Steps

latest

### Optional WorldGuard-Report

_No response_

### Anything Else?

_No response_

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -27,7 +27,6 @@ junit-jupiter-api.module = "org.junit.jupiter:junit-jupiter-api"
 junit-jupiter-params.module = "org.junit.jupiter:junit-jupiter-params"
 junit-jupiter-engine.module = "org.junit.jupiter:junit-jupiter-engine"
 
-spigot = "org.spigotmc:spigot-api:1.21.4-R0.1-SNAPSHOT"
 paperApi = "io.papermc.paper:paper-api:1.21.4-R0.1-SNAPSHOT"
 paperLib = "io.papermc:paperlib:1.0.8"
 

--- a/worldguard-bukkit/build.gradle.kts
+++ b/worldguard-bukkit/build.gradle.kts
@@ -5,23 +5,10 @@ plugins {
     id("buildlogic.platform")
 }
 
-val localImplementation = configurations.create("localImplementation") {
-    description = "Dependencies used locally, but provided by the runtime Bukkit implementation"
-    isCanBeConsumed = false
-    isCanBeResolved = false
-}
-configurations["compileOnly"].extendsFrom(localImplementation)
-configurations["testImplementation"].extendsFrom(localImplementation)
-
 dependencies {
     "api"(project(":worldguard-core"))
     "api"(libs.worldedit.bukkit) { isTransitive = false }
     "compileOnly"(libs.commandbook) { isTransitive = false }
-    // Technically this is api, but everyone should already have some form of the bukkit API
-    // Avoid pulling in another one, especially one so outdated.
-    "localImplementation"(libs.spigot) {
-        exclude("junit", "junit")
-    }
 
     "compileOnly"(libs.jetbrains.annotations) {
         because("Resolving Spigot annotations")

